docs/examples/model_ex/model_all_comparisons_no_model_unc.py

```diff
@@ -1,8 +1,8 @@
 """
-All data/model comparisons, no model uncertainty
-================================================
+Data/model comparisons, no model uncertainty
+============================================
 
-This example shows every possible comparison between data and model, without model uncertainty.
+Multiple comparisons between data and model, without model uncertainty.
```

Comment:
Why omit the fact that it is indeed all the possible comparisons that make sense with `plot_data_model_comparison()`?